### ISSUE TITLE
work with production mode

### ISSUE THREAD

**Issue by onlinebizsoft** [ho-nl/magento2-Ho_Templatehints#55]
I'm not sure if it is possible technically but it would be great if we can run this with production mode

**Comment by Eddcapone** [comment]
No, this makes no sense. You should never develop in production mode...

**Comment by PascalBrouwers** [comment]
Nothing wrong with testing your code also in production mode. 
@onlinebizsoft did you experience any issue with running Ho_Templatehints with production mode?

**Comment by Eddcapone** [comment]
@PascalBrouwers This is not a tool which is about testing and it is not good for performance.

**Comment by onlinebizsoft** [comment]
@Eddcapone "production mode" can be enabled on many environment (data entry site, content-preparation site, QA site, stage site,....etc), not only the production site. So I don't see any reason why we need to restrict this if users still want to enable it on in the production mode
@PascalBrouwers it doesn't work if production mode is enabled (doesn't show any information)

**Comment by PascalBrouwers** [comment]
Hmm ok, then we need to look into it.

**Comment by Eddcapone** [comment]
> 

Because it is not best practice. You should have a development instance and a production one.

**Comment by onlinebizsoft** [comment]
@Eddcapone we have separated instances but I mean on some instances we may set production mode for using.

**Comment by Eddcapone** [comment]
@onlinebizsoft Then activate it on the development instance ^^ why do you need it in production?

**Comment by PascalBrouwers** [comment]
@onlinebizsoft just ignore him. It's a valid point and we'll look into it why it's not working during production mode
